paper/paper.md

```diff
@@ -85,20 +83,6 @@ using FluxNLPModels
 LeNetNLPModel = FluxNLPModel(LeNet, train_loader, test_loader; loss_f = Flux.logitcrossentropy)
 ```
 
-Or to build a Knet.jl architecture, you must define the layers you need: convolutional and dense layers and link them properly.
```

Comment:
are you sure that we don't need to show how KnetNLPModels  is used ?

Reply:
Yes